portfolio/src/main/java/com/google/sps/data/Nickname.java

```diff
@@ -0,0 +1,27 @@
+package com.google.sps.data;
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.Query.SortDirection;
+
+public final class Nickname{
+    /**
+   * Returns the nickname of the user with id, or empty String if the user has not set a nickname.
+   */
+  public String getUserNickname(String id) {
```

Comment:
If you don't need instances of this class for anything, you can make the method static and make ithis class impossible to construct by declaring an empty private constructor.

Reply:
Ahh I always get errors when calling a non-static method from a static so I assumed it was the same vice versa and made it a public method. But yeah, static just belongs to the class, my bad.